feature/afts/otg_tests/afts_base/afts_base_test.go

```diff
@@ -169,12 +170,19 @@ func (tc *testCase) configureDUT(t *testing.T) error {
 	}
 	dutConf := createBGPNeighbor(peerGrpNameV4P1, peerGrpNameV6P1, nbrs, dut)
 	gnmi.Update(t, dut, dutConfPath.Config(), dutConf)
+	if deviations.BgpNeighborPrefixLimitRequired(dut) {
```

Comment:
The preferred style would be to refactor the bgp config in this test to use the bgp config generation in cfgplugins/bgp.go .

https://github.com/openconfig/featureprofiles/blob/main/internal/cfgplugins/bgp.go#L161

 Then embed this deviation into the bgp neighbor configuration inside that plugin here:

https://github.com/openconfig/featureprofiles/blob/6b6d553b3fa31e2ae3af8622883e4b4a3167b41a/internal/cfgplugins/bgp.go#L245

Reply:
Using the style here, one would need to sweep all the tests in the entire repo that configure bgp prefix limits and add the deviation over and over again.

I did a little search for use of the OC max-prefixes and I see a deviation has already been defined for this at:
https://github.com/openconfig/featureprofiles/blob/6b6d553b3fa31e2ae3af8622883e4b4a3167b41a/feature/gribi/otg_tests/fib_failed_due_to_hw_res_exhaust_test/fib_failed_due_to_hw_res_exhaust_test.go#L267

Duplicating deviations  is another reason not to embed them, one-off, into individual tests.  Rather use a common config plugin to generate the piece of config you want.  If that needs a deviation, then put it in the plugin so all tests will benefit from it.